.ci/magician/cmd/reassign_reviewer.go

```diff
@@ -0,0 +1,126 @@
+/*
+* Copyright 2024 Google LLC. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+ */
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"magician/github"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// reassignReviewerCmd represents the reassignReviewer command
+var reassignReviewerCmd = &cobra.Command{
+	Use:   "reassign-reviewer PR_NUMBER [REVIEWER]",
+	Short: "Reassigns primary reviewer to the given reviewer or a random reviewer if none given",
+	Long: `This command reassigns reviewers when invoked via a comment on a pull request.
+
+	The command expects the following PR details as arguments:
+	1. PR_NUMBER
+	2. REVIEWER (optional)
+
+
+	It then performs the following operations:
+	1. Updates the reviewer comment to reflect the new primary reviewer.
+	2. Requests a review from the new primary reviewer.
+	`,
+	Args: cobra.MinimumNArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		prNumber := args[0]
+		fmt.Println("PR Number: ", prNumber)
+
+		githubToken, ok := os.LookupEnv("GITHUB_TOKEN_MAGIC_MODULES")
```

Comment:
I think this should just use GITHUB_TOKEN since it'll run in the context of a GHA.

```suggestion
		githubToken, ok := os.LookupEnv("GITHUB_TOKEN")
```

Reply:
Done